### DISCUSSION TITLE
Setup Manual with external Postgresql DB

### DISCUSSION THREAD

**Discussion by manuelwallrapp** [tektoncd/results#369] [Ideas]
Hi there

The setup manual describes how to setup the tekton-results with a postgresql stateful set withtin the openshift cluster. It would be also cool to describe how to set it up with an external Postgresql DB. I set it up with an external DB. Would have saved me a little time to have it in the manual. What do you think?

Best regards

Manu

**Comment by alan-ghelardi** [comment]
I just created https://github.com/tektoncd/results/issues/371 to keep track of this improvement on the documentation.

**Comment by adambkaplan** [comment]
I'm going to close this discussion, since we have effectively "promoted" it to a feature request. Thanks!
